packages/react/src/components/ImageComponent/ImageComponent.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ *  sorts media query min-widths order to ensure the
+ * browser returns the proper sources and the specified widths
+ *
+ * @param {Array} sources image sources and min-widths
+ *
+ * @returns {Array} sorted array of sources
+ */
+const sortSources = sources => {
+  return sources.sort((a, b) => (a.minWidth > b.minWidth ? -1 : 1));
+};
+
+/**
+ * renders background image
+ *
+ * @param {object} props props object
+ * @param {object} props.images array of images used for diff breakpoints
+ * @param {string} props.defaultImage default image (usually image for largest breakpoint)
+ * @param {string} props.alt alt of the image
+ * @returns {*} picture element
+ */
+const Image = ({ images, defaultImage, alt }) => {
+  const sortedImages = sortSources(images);
+
+  return (
+    <picture data-autoid={`${stablePrefix}--imagecomponent`}>
```

Comment:
I don't think we need the word "component" in the name of the component. I'm double-checking with @wonilsuhibm to see if we aligned on a name for this yet. I threw out just `Picture`, or  I think `ResponsivePicture` was suggested, although it seems like there are bigger plans for this component. I think @wonilsuhibm suggested `EmbeddedPicture`. I kind of either like just the simple `Picture`, or maybe a `RichPicture` because I think there might be more features to come on this down the road.

Reply:
Agreed, `component` is probably not necessary here.

Reply:
@jeffchew @photodow Should i update that to RichPicture or Embedded picture or Just picture something like that?

Reply:
still waiting for the suggestions

Reply:
Let's go with `Picture`, thoughts?

Reply:
Yeah, let's go with `Picture`. Sorry, I was working to get a response from Wonil, but I guess he's been too busy to respond.

Reply:
@jeffchew @carthick I finally heard back from Wonil today. Could we go with `Image` instead? Wonil was worried `Picture` sounds too much like something like a product image, and doesn't account for things like charts and such. It has a previous meaning to some of the other designers.

Reply:
NP @photodow I have updated that to Image ....